app/lib/mappers/mapNonCoreElements.js

```diff
@@ -0,0 +1,34 @@
+const utils = require('../utils');
+
+function getElements(nonCoreElements) {
+  return nonCoreElements && nonCoreElements.nonCoreElement ?
+    utils.asArray(nonCoreElements.nonCoreElement) : [];
+}
+
+function getElement(nonCoreElements, title) {
+  const element = getElements(nonCoreElements).find(el => el.elementTitle === title);
+  return element ? element.elementText : undefined;
```

Comment:
Can `elementText` be trimmed please? I've noticed trailing spaces in the content and i don't think it needs to be there.